lib/routes/authorization.js

```diff
@@ -160,6 +161,12 @@ module.exports = {
           is: CODE,
           then: Joi.optional(),
           otherwise: Joi.forbidden()
+        }),
+      keys_jwe: Joi.string()
```

Comment:
Add length restriction here?

Reply:
We could also validate that it's only b64 encoded characters, or whatever format restrictions we can validate cheaply.